src/WebJobs.Script/Utility.cs

```diff
@@ -520,8 +520,28 @@ private static bool ContainsFunctionWithWorkerRuntime(IEnumerable<FunctionMetada
             return false;
         }
 
-        public static bool CheckAppOffline(string scriptPath)
+        public static async Task MarkContainerDisabled()
```

Comment:
Can you write a log statement when this happens?

Reply:
Fixed.